sqlglot/dialects/snowflake.py

```diff
@@ -183,6 +185,26 @@ class Parser(parser.Parser):
             TokenType.PARTITION_BY: lambda self: self._parse_partitioned_by(),
         }
 
+        def _parse_except(self) -> t.Optional[t.List[t.Optional[exp.Expression]]]:
+            """
+            Override except parser to support Snowflake syntax without parens
+            """
+            if not self._match(TokenType.EXCEPT):
+                return None
+            if self._match(TokenType.L_PAREN, advance=False):
```

Comment:
sqlglot tries to make the default parser as leniant as possible, can we move this to the base parser?

Reply:
additionally, you're going to need to have a customer generator for snowflake to properly output EXCLUDE instead of EXCEPT